rllib/utils/test_utils.py

```diff
@@ -1392,15 +1392,18 @@ def run_rllib_example_script_experiment(
         # Define compute resources used automatically (only using the --num-gpus arg).
         # New stack.
         if config.enable_rl_module_and_learner:
+            # Do we have GPUs available in the cluster?
+            num_gpus = ray.cluster_resources().get("GPU", 0)
+            if args.num_gpus > 0 and num_gpus < args.num_gpus:
+                raise ValueError(
+                    f"You are running your script with --num-gpus={args.num_gpus}, "
+                    "but your cluster only has {resources['GPU']} GPUs! Will "
```

Comment:
Something missing? "Will ... "?

Reply:
Fixed. Now, it'll just warn and then run with CPU-based Learners instead.